### ISSUE TITLE
Xattrs do not work

### ISSUE THREAD

**Issue by peesock** [netheril96/securefs#188]
I can't get xattrs to work, either with `overlayfs` which uses xattrs to track filesystem changes, or directly with `getfattr`:
```
$ cd mount
$ touch file
$ getfattr file
getfattr: file: Operation not supported
```
Tested with lite and full formats with default parameters, under both btrfs and tmpfs backing filesystems.
`--trace` doesn't log anything when using getfattr.

**Comment by netheril96** [comment]
Currently `securefs` only supports xattr on macOS, due to the extensive usage of xattr on this OS. Are you trying on Linux?

If you have serious needs for xattr on Linux, rather than just toy with it, please let me know. I'll implement it in the following versions.

**Comment by peesock** [comment]
Yes I'm on linux, I thought the docs meant macOS has special handling of xattrs, not that only macOS supported them, sorry.

My use is to mount compressed read-only filesystems and add a writable layer with `overlayfs`, which requires xattrs to track things like deleted files. 

For now, i'm migrating those parts of my `securefs` mount to `gocryptfs`, as they aren't very fingerprintable and thus don't need the extra security. Thanks for considering support!

**Comment by netheril96** [comment]
So the layering is securefs/gocryptfs ➡️compressed readonly filesystem➡️overlayfs?

**Comment by peesock** [comment]
Yes

**Comment by netheril96** [comment]
Fixed in https://github.com/netheril96/securefs/releases/tag/v1.1.0
